### PR TITLE
Add very basic end to end testing

### PR DESCRIPTION
**Pull request by joewalker**
Also adds a quick test mode so this doesn't get out of hand.

Signed-off-by: Joe Walker <jwalker@mozilla.com>

### CODE REVIEW

**Comment by joewalker** [comment]
f? @mossop

File: test/webdriver/test-basic.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+describe('application launch', function() {
+  if (quickTest) {
+    it.skip('all tests');
+    return;
+  }
+
+  this.timeout(10000);
+
+  beforeEach(async function() {
+    await Tasks.build();
```
Comment (Mossop) [inline test/webdriver/test-basic.js]: Why not do this before starting the tests rather than before each test?

Comment (joewalker) [inline test/webdriver/test-basic.js]: Fixed in f07f1cf

**Comment by victorporof** [comment]
Can someone explain to me the reasoning behind this PR?

**Comment by Mossop** [comment]
We want to have tests that verify that the app actually starts up

**Comment by joewalker** [comment]
Closing in preference for #230.

**Pull request closed without merging**